app/models/post.rb

```diff
@@ -0,0 +1,24 @@
+class Post < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  # Attributes
+  attribute :title, :string
+  attribute :text, :text
+  attribute :comments_counter, :integer, default: 0
+  attribute :likes_counter, :integer, default: 0
+
+  # Callback
+  after_save :update_user_posts_counter
+
+  # Methods
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count + 1)
+  end
```

Comment:
- [ ] [OPTIONAL]
Following best practices `update_posts_counter` should be marked as _private_ after implementing the `after_save` method.